### ISSUE TITLE
commonjs module exports

### ISSUE THREAD

**Issue by BurningEnlightenment** [arantes555/electron-fetch#7]
In `v1.1.0` the exports of index.js look like:
```js
exports = fetch;
exports.Headers = Headers;
exports.Request = Request;
exports.Response = Response;
exports.FetchError = FetchError;
```

In `v1.2.x` however it was changed to:
```js
exports.default = fetch;
exports.Headers = Headers;
exports.Request = Request;
exports.Response = Response;
exports.FetchError = FetchError;
```
which is incompatible with the current typings. Was this intended?

**Comment by arantes555** [comment]
@BurningEnlightenment Nope, this was not intended, and reviewing the changes I can't see what would have caused it. I did bump a few dependencies, but nothing that should have produced such a change. I did not even notice because I always use the es6 module...

EDIT: After a bit of investigation, looks like it was always supposed to behave as it currently does. I guess the previous behaviour was a bug in the old rollup version, possibly https://github.com/rollup/rollup/pull/2001 . I'll be happy to accept a fix in the typings
